TestHelpers.Tests/MockFileCopyTests.cs

```diff
@@ -191,6 +213,50 @@ public void MockFile_Copy_ShouldThrowNotSupportedExceptionWhenTargetFileNameCont
             }
         }
 
+        [Test]
+        [WindowsOnly(WindowsSpecifics.Drives)]
+        public void MockFile_Copy_ShouldThrowNotSupportedExceptionWhenSourcePathContainsInvalidUseOfDriveSeparator()
+        {
+            var badSourcePath = XFS.Path(@"C::\something\demo.txt");
+            var destinationPath = XFS.Path(@"C:\elsewhere\demo.txt");
+            var fileSystem = new MockFileSystem();
+
+            Assert.Throws<NotSupportedException>(() => fileSystem.File.Copy(badSourcePath, destinationPath));
```

Comment:
Small nit, but it's generally preferred to split up the Act and Assert steps.

```csharp
TestDelegate action = () => fileSystem.File.Copy(badSourcePath, destinationPath);

Assert.Throws<NotSupportedException>(action);
```

Makes the debugging and readability story a bit better.

Reply:
Not my style, but sure, done.